.github/workflows/label-actions.yml

```
@@ -0,0 +1,21 @@
+name: 'Label Actions'
+
+on:
+  issues:
+    types: [labeled, unlabeled]
+  pull_request_target:
+    types: [labeled, unlabeled]
+  discussion:
+    types: [labeled, unlabeled]
+
+permissions:
+  contents: read
+  issues: write
+  pull-requests: write
+  discussions: write
+
+jobs:
+  action:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: dessant/label-actions@v4
```

As this is a third-party action and we are requiring extra permissions that could have potential security implications, we should pin to commit sha similar to: https://github.com/docker/buildx/blob/8afeb56a3b00d94b86a98045b5b43f8bba94faad/.github/workflows/build.yml#L359

```suggestion
      - uses: dessant/label-actions@102faf474a544be75fbaf4df54e73d3c515a0e65  # v4.0.1
```

Would also need to audit this action to make sure there is nothing wrong with it.